src/components/skip-link/index.md.njk

```diff
@@ -15,7 +15,9 @@ Use the skip link component to help keyboard-only users skip to the main content
 
 ## When to use this component
 
-All GOV.UK pages must include a skip link in the header.
+All GOV.UK pages must include a skip link, placed on its own at the top of the `<body>` tag, and outside of the `<header>' tag.
```

Comment:
Is 'must' right here? Would 'should' be better (if there are implementations that differ from what we're recommending)?

Reply:
In theory, it has relatively little value if a page has no navigation – as it would 'only' be skipping the GOV.UK logo in the header.

However I think we have historically always told teams to include one, so that we're consistent across the site.

So yes, I think 'must' is correct.

Reply:
Thanks @36degrees!